gamemodes/terrortown/gamemode/server/sv_ent_replace.lua

```diff
@@ -382,12 +386,15 @@ end
 -- entities of the classes given the table
 local function PlaceWeaponsAtEnts(spots_classes)
 	local spots = {}
+	local count = 0
 
 	for i = 1, #spots_classes do
 		local entsTbl = ents.FindByClass(spots_classes[i])
 
 		for k = 1, #entsTbl do
-			spots[#spots + 1] = entsTbl[k]
+			count = count + 1
+
+			spots[count] = entsTbl[k]
```

Comment:
does this really have any benefit?

Reply:
Yes. No table call but direct access, if not optimized. Also seen here (without any speedtests sadly): https://stigmax.gitbook.io/lua-guide/auxiliary/optimizations#adding-values-to-tables

Reply:
alrighht, good to know!